hypervisor/src/mshv/mod.rs

```diff
@@ -1495,17 +1505,16 @@ impl MshvVcpu {
     /// Clear SW_EXIT_INFO1 register for SEV-SNP guests.
     ///
     #[cfg(feature = "sev_snp")]
-    fn clear_swexit_info1(
-        &self,
-        ghcb_gpa: u64,
-    ) -> std::result::Result<cpu::VmExit, cpu::HypervisorCpuError> {
+    fn clear_swexit_info1(&self) -> std::result::Result<cpu::VmExit, cpu::HypervisorCpuError> {
         // Clear the SW_EXIT_INFO1 register to indicate no error
-        self.gpa_write(ghcb_gpa + GHCB_SW_EXITINFO1_OFFSET, &[0; 4])?;
+        let ghcb = self.ghcb.as_ref().unwrap().0;
+        set_svm_field_u64_ptr!(ghcb, exit_info1, 0);
 
         Ok(cpu::VmExit::Ignore)
     }
 
     #[cfg(feature = "sev_snp")]
+    #[allow(dead_code)]
     fn gpa_read(&self, gpa: u64, data: &mut [u8]) -> cpu::Result<()> {
```

Comment:
@russell-islam lets remove this (and `gpa_write()` too if possible)

Reply:
gpa_write is still used, I added a comment for a future work. Lets keep the gpa_write atm.